src/riscv-integration.adoc

```diff
@@ -330,7 +330,7 @@ two tag bits in memory.
 
 {cheri_base_ext_name} requires that RISC-V CSRs intended to hold addresses,
 like <<mtvec>>, are now able to hold capabilities. Therefore, such registers are
-renamed and extended to CLEN-bit in {cheri_base_ext_name}.
+renamed and extended to CLEN+1 bits in {cheri_base_ext_name}.
```

Comment:
Perhaps something like
> for brevity, this document simply says "The Xc register extends X" rather than repeatedly pointing out the simultaneous renaming.

Reply:
Thatnks that sounds good.

Reply:
You are right, the "rename" idea was from a previous iteration of the spec when we had separate addresses for the Xc CSRs. IMHO, this chapter makes the spec confusing because it still talks about "rename", so I suggest removing that altogether and just talk about extended registers.

Also, note that at this point in the spec we are defining Zcheripurecap which does not support the CHERI mode, so in fact, a CPU only implementing this extension would only ever have access to the "extended" Xc registers.